resources/web/streaming_viewer/index.html

```diff
@@ -11,18 +11,33 @@
   </head>
 
   <body>
-    <div style="padding-left:10px;height:100px">
+    <header>
       <h1>Webots streaming viewer</h1>
-      <p> Connect to: <input id="IPInput" type="text" value="localhost" /> : <input id="PortInput" type="text" value="1234" maxlength="5" size="5" /> mode:
+
+      <p> Connect to: <input id="IPInput" type="text" size="60" value="ws://localhost:1234" />
```

Comment:
Note that we should apply the same changes also to resources/web/streaming_viewer/index-debug.html
The only difference between the two files is that one is using the minified version.

Reply:
I missed your comment :( I will create another PR to fix it

Reply:
No problem.

It's already done here #2261